src/components/TxData.js

```diff
@@ -512,6 +508,7 @@ class TxData extends React.Component {
 
     return (
       <div>
+        <TxAlerts tx={this.props.transaction} />
```

Comment:
Commenting here to create a discussion thread.

I like the code in general but I think I don't like having the alert as an info. I believe when the tx has only info metadata, we should show only the marker, which I like a lot and leave only the success alert with 'This block/tx is valid'.

For any banned/scam txs I think we can add the red alert for more attention.

Reply:
> I believe when the tx has only info metadata, we should show only the marker

Like an `i` icon with the "this is the 1,000,000th" text on hover?

Reply:
No, I mean only the TxMarker you created
![image](https://user-images.githubusercontent.com/3298774/125969920-96c50257-2f8c-493a-9e7b-a813f0242517.png)

Reply:
I agree that the alerts must be different for each type. I think we should only use a whole line in the screen for specific alerts such as banned/scamming transactions.